habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -244,6 +244,21 @@ class BaseVelocityActionConfig(ActionConfig):
     allow_back: bool = True
 
 
+@dataclass
+class BaseVelocitySpotActionConfig(ActionConfig):
+    r"""
+    In Rearrangement only. Corresponds to the base velocity. Contains two continuous actions, the first one controls forward and backward motion, the second the rotation.
+    """
+    type: str = "BaseVelSpotAction"
+    lin_speed: float = 10.0
```

Comment:
silly question, but is this meant to be the max_lin_speed? If we can't rename the variable, can we add a comment clarifying this?

Reply:
Yes, thank you! I added the comments